build.gradle

```diff
@@ -744,7 +744,9 @@ task smoke() {
 
     finalizedBy {
         generateCucumberReports {
-            doLast{
+            doLast {
+                delete "${rootDir}/BEFTA Report for Smoke Tests/"
```

Comment:
The directory name `BEFTA Report for Smoke Tests` is re-used throughout this file, which would be a pain to update if it changed - be better as a constant ?